db/migrate/20180226171602_add_index_to_job_execution_job_queue_id_and_status.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToJobExecutionJobQueueIdAndStatus < ActiveRecord::Migration[5.1]
+  def change
+    add_index :barbeque_job_executions, [:job_queue_id, :status]
```

Comment:
Is this index really needed?
barbeque_job_executions already has index on status column and it's enough to select running job executions.
https://github.com/cookpad/barbeque/pull/61/files#diff-1ab0daf75dcc12148205a09acc34dc80L48

Reply:
I was thinking of dropping index on status column in https://github.com/cookpad/barbeque/pull/62, but this would not be useful unless we have many queues. Removed in 360817a.